src/mapper/components/HeaderMapperSelection.tsx

```diff
@@ -42,8 +42,8 @@ export default function HeaderMapperSelection({
                 />
               </svg>
             </div>
-          </Col>
-          <Col>
+          </div>
+          <div className="flex-1">
```

Comment:
Isn't there something wrong on this one? To me looks like name and dropdown aren't aligned vertically?
<img width="795" alt="Screenshot 2025-02-14 at 8 02 21 AM" src="https://github.com/user-attachments/assets/08e30d32-bd2c-4e31-b2ca-4f6c5d62c126" />

Reply:
I think Select component might be adding an unnecessary top padding?

Reply:
Plus we might want to add `items-center` to the top div?

Reply:
<img width="854" alt="Screenshot 2025-02-14 at 8 12 21 AM" src="https://github.com/user-attachments/assets/207d777c-9a21-45e8-b16a-f15e91650721" />

Something like this (we probably don't need the inner div, so I removed it)